tests/ffi/dev_mem_inaccessible/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+. ../../e2e/lib/utils
+
+dev_mem_inaccessible(){
+    expected_result="ls: cannot access '/dev/mem': No such file or directory"
```

Comment:
why don't we just check if the command fail? Checking string output from commands are not safe IMHO.

Reply:
Yes, you are right. But just checking if the command failed, I'm not sure if that's enough to tell /dev/mem doesn't exist.